scripts/postinstall.js

```diff
@@ -0,0 +1,38 @@
+var semver = require('semver');
+
+var NODE_MIN_VER_WITH_BUILTIN_SCRYPT = '10.5.0';
+var NODE_MIN_VER_INCOMPAT_SCRYPT_PKG = '12.0.0';
+// there are presently no Node.js versions with built-in scrypt that doesn't
+// have maxmem limitations
+// see: https://github.com/nodejs/node/pull/28799#issuecomment-521829488
+// anticipating improved scrypt in 10.x and 12.x release lines
+// var NODE_MIN_VER_SCRYPT_MAXMEM_FIXED = ['10.?.?', '12.?.?'];
+
+// use NODE_MIN_VER_SCRYPT_MAXMEM_FIXED instead, when such versions are available
+// var byUpgradingOrInstalling = 'You can improve the performance of web3-eth-accounts by upgrading to Node.js version ' + NODE_MIN_VER_SCRYPT_MAXMEM_FIXED.join(' or ') + ' or newer, or by installing the (deprecated) scrypt package in your project';
+
+var byUpgradingOrInstalling = 'You can improve the performance of web3-eth-accounts by upgrading to Node.js version ' + NODE_MIN_VER_WITH_BUILTIN_SCRYPT + ' or newer, or by installing the (deprecated) scrypt package in your project';
```

Comment:
One more swap to `scrypt` needed here, my apologies for forgetting to point it out in my previous review.